queue/db.go

```diff
@@ -0,0 +1,22 @@
+package queue
+
+import (
+	"context"
+	"database/sql"
+)
+
+type DBTX interface {
```

Comment:
not sure i get what this file is for

Reply:
is this something sqlc specific? could use a comment if so

Reply:
Yeah this is auto-generated. Will leave it for now